config/default/kustomization.yaml

```diff
@@ -1,63 +0,0 @@
-# Adds namespace to all resources.
```

Comment:
This file configures all related yaml manifests of this project operator, including CRDs, RBAC objects and Deployment,
it will patch the namespace and all objects names with the names specified in this file (namespace - line 2, name-prefix - line 9).

Please keep this file and remove the commented lines related to webhook and certmanager.

Reply:
done.